### PR TITLE
Spike/hec/dry events

### PR DESCRIPTION
**Pull request by fgregg**

### CODE REVIEW

File: legistar/events.py [hunk 1]
```diff
@@ -138,6 +138,29 @@ def ical(self, ical_text):
...
+            webscraper.cache_storage = self.cache_storage
+
+        if self.requests_per_minute == 0:
+            webscraper.cache_write_only = False
+
+        webscraper.BASE_URL = self.WEB_URL
+
+        return webscraper
+
+    def _get_web_event(self, api_event):
```
Comment (fgregg) [inline legistar/events.py]: nice error! Might also be an opportunity to an AbstractBaseClass? https://stackoverflow.com/questions/3570796/why-use-abstract-base-classes-in-python

Up to you.

**Comment by fgregg** [comment]
I approve this PR. Nice factoring!

File: legistar/events.py (hunk 1 above)
Comment (fgregg) [reply legistar/events.py]: @hancush ^

Comment (hancush) [reply legistar/events.py]: Oh boy, I like that! The long-term maintainability argument [here](https://dbader.org/blog/abstract-base-classes-in-python) was very compelling. Switched!

**Pull request merged**